x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -188,6 +188,7 @@ export const policyFactoryWithoutPaidFeatures = (
 
   return {
     ...policy,
+    global_manifest_version: 'latest',
```

Comment:
This is used for gold and below licenses it seems. What happens with Platinum ones? Are we gonna modify other things here allowed on Platinum licenses?

Reply:
For anything but `enterprise` I believe we want to set `global_manifest_version` always to `latest`, i.e. we are not allowing user to set a custom date. As above, thats how I understand all this ;)

Reply:
Suggestion to do something cleaner here:

Can we create a similar factory to this that is specific to `enterprise` features?

Then we can leave this line alone as it actually should be run when license is at least platinum: https://github.com/elastic/kibana/pull/168113/files#diff-6a6ff8e750469752a4993b278b7023af4bef43a49ccee59c3933e3d7c68c78f5R286

Then for this line, we can use the new function that only changes `global_manifest_version` when we drop below `enterprise`.  That way we can keep all existing logic the same and just introduce a new function that will replace the one field we care about.